static/js/admin/vue-cdr-admin/src/components/chompb/preIngest.vue

```diff
@@ -0,0 +1,175 @@
+<!--
+Injects chompb dashboard into web-admin-app/src/main/webapp/WEB-INF/jsp/report/chompb.jsp
+without having to include the vue-admin js app again. It does this via Vue's teleport option
+https://vuejs.org/guide/built-ins/teleport.html
+-->
+<template>
+    <teleport to="#chompb-admin">
+        <div id="chompb-preingest-ui">
+            <h2 class="chompb-ui has-text-weight-semibold is-size-3 has-text-centered">Pre-ingest Projects</h2>
+            <data-table @click="copyPath($event)" id="chompb-projects" class="table is-striped is-bordered is-fullwidth"
+                        :data="dataSet"
+                        :columns="columns"
+                        :options="tableOptions">
+                <thead>
+                <tr>
+                    <th>Chompb Project</th>
+                    <th>Source</th>
+                    <th>Status</th>
+                    <th>Actions</th>
+                </tr>
+                </thead>
+            </data-table>
+            <div id="copy-msg" class="notification is-light" :class="copyMsgClass" v-if="copy_msg !== ''">{{ copy_msg }}</div>
+        </div>
+    </teleport>
+</template>
+
+<script>
+import DataTable from 'datatables.net-vue3';
+import DataTablesLib from 'datatables.net-bm';
+
+DataTable.use(DataTablesLib);
+
+export default {
+    name: 'preIngest',
+
+    components: {DataTable},
+
+    data() {
+        return {
+            copy_error: false,
+            copy_msg: '',
+            columns: [
+                { data: 'Chompb Project' },
+                { data: 'Source' },
+                { data: 'Status' },
+                { data: 'Actions' }
+            ],
+            dataSet: []
+        }
+    },
+
+    computed: {
+        /* ajaxOptions() {
```

Comment:
while i think we are going to need ajax options at some point, we probably aren't going to need this search result retrieval logic. Maybe either remove this whole block, or at least remove all the search result stuff to avoid confusion?